File: JavaScript/01-Number-Programs/fibonacci.js
function fibonacciSeries(num) {
  let first = 0;
  let second = 1;

  console.log(first);
  console.log(second);

  for (let i = 2; i <= num; i++) {
    let next = first + second;
    console.log(next);
    first = second;
    second = next;
  }
}
fibonacciSeries(8);

File: 01-Number-Programs/01-Number-Programs/Fibonacci.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _01_Number_Programs
{
    internal class Fibonacci
    {
        void FibonacciSeries(int num)
        {
            int first = 0;
            int second = 1;
            Console.WriteLine(first);
            Console.WriteLine(second);

            for(int i = 2; i<=num; i++)
            {
                int next = first + second;
                Console.WriteLine(next);
                first = second;
                second = next;
            }
        }

        public static void Main(string[] args)
        {
            Fibonacci f = new Fibonacci();
            f.FibonacciSeries(5);
        }
    }
}
